src/android/java/com/mattrayner/vuforia/app/ImageTargets.java

```diff
@@ -57,40 +57,41 @@
 public class ImageTargets extends Activity implements ApplicationControl
 {
     private static final String LOGTAG = "ImageTargets";
-    
+    private static final String FILE_PROTOCOLE_FIX = "file://";
+
```

Comment:
`FILE_PROTOCOL_PREFIX` is also fine if you like - whatever seems clearer. Constants for the iOS part would be good but that's up to you, otherwise just this fix and it can be merged 👍